views/designerHTML.ejs

```diff
@@ -1,3 +1,17 @@
+<div class="deprecation" id="deprecation">
+  <a class="closeWarning" id="closeWarning">&times;</a>
+  Our tools for teaching the Web are evolving. As of September 30th, Mozilla will no longer support Appmaker. To learn more, read  <a href="http://mzl.la/webmaker-tools-transition" target="_blank">our blog post.</a>
+  <script>
+    (function () {
+      var closeWarning = document.querySelector('#closeWarning');
+      closeWarning.onclick = function (e) {
+        closeWarning.parentNode.remove();
```

Comment:
technically, `remove()` is not a function in the JS DOM API (it's available in some consoles, which shim some jQuery-esque functions in). If vanilla JS, to remove the closeWarning, this should be `closeWarning.parentNode.removeChild(closeWarning)`. If the intention is to remove the closeWarning's parent, let's give that the `id` instead, and then remove it directly.